openff/utilities/provenance.py

```diff
@@ -10,21 +9,28 @@ def _get_conda_list_package_versions() -> Dict[str, str]:
     from openff.utilities.exceptions import CondaExecutableNotFoundError
     from openff.utilities.utilities import has_executable
 
-    if has_executable("mamba"):
+    if has_executable("micromamba"):
+        conda_executable = "micromamba"
+    elif has_executable("mamba"):
         conda_executable = "mamba"
     elif has_executable("conda"):
         conda_executable = "conda"
-    elif has_executable("micromamba"):
-        conda_executable = "micromamba"
```

Comment:
`setup-micromamba` brings in a `conda` executable for reasons that are opaque to me and exceedingly difficult to test. This change is arguably an improvement but is mostly there to ensure the problematic behavior can be reproduced.

Reply:
🎉